code/src/gfx.c

```diff
@@ -617,17 +622,33 @@ static void Gfx_DrawERTrackerGroups(void) {
         u32 locPosY = listTopY + ((SPACING_SMALL_Y + 1) * entrance * 2);
         u32 itemPosY = locPosY + SPACING_SMALL_Y;
 
-        bool isDiscovered = IsEntranceDiscovered(rEntranceOverrides[locIndex].index);
+        if (!destListToggle) {
+            bool isDiscovered = IsEntranceDiscovered(rEntranceOverrides[locIndex].index);
 
-        u32 color = isDiscovered ? COLOR_GREEN : COLOR_WHITE;
-        const char* unknown = "???";
-        const char* origSrcName = GetEntranceName(rEntranceOverrides[locIndex].index);
-        const char* origDstName = GetEntranceName(rEntranceOverrides[locIndex].destination);
-        const char* rplcSrcName = gSettingsContext.ingameSpoilers || isDiscovered ? GetEntranceName(rEntranceOverrides[locIndex].override) : unknown;
-        const char* rplcDstName = gSettingsContext.ingameSpoilers || isDiscovered ? GetEntranceName(rEntranceOverrides[locIndex].overrideDestination) : unknown;
+            u32 color = isDiscovered ? COLOR_GREEN : COLOR_WHITE;
+            const char* unknown = "???";
 
-        Draw_DrawFormattedString_Small(10, locPosY, color, "%s to %s %c", origSrcName, origDstName, RIGHT_ARROW_CHR);
-        Draw_DrawFormattedString_Small(10, itemPosY, color, "  %s from %s", rplcDstName, rplcSrcName);
+            const char* origSrcName = GetEntranceName(rEntranceOverrides[locIndex].index);
+            const char* origDstName = GetEntranceName(rEntranceOverrides[locIndex].destination);
+            const char* rplcSrcName = gSettingsContext.ingameSpoilers || isDiscovered ? GetEntranceName(rEntranceOverrides[locIndex].override) : unknown;
+            const char* rplcDstName = gSettingsContext.ingameSpoilers || isDiscovered ? GetEntranceName(rEntranceOverrides[locIndex].overrideDestination) : unknown;
+
+            Draw_DrawFormattedString_Small(10, locPosY, color, "%s to %s %c", origSrcName, origDstName, RIGHT_ARROW_CHR);
+            Draw_DrawFormattedString_Small(10, itemPosY, color, "  %s from %s", rplcDstName, rplcSrcName);
+        } else {
+            bool isDiscovered = IsEntranceDiscovered(destList[locIndex].index);
+
+            u32 color = isDiscovered ? COLOR_GREEN : COLOR_WHITE;
+            const char* unknown = "???";
+
+            const char* origSrcName = GetEntranceName(destList[locIndex].index);
+            const char* origDstName = GetEntranceName(destList[locIndex].destination);
+            const char* rplcSrcName = gSettingsContext.ingameSpoilers || isDiscovered ? GetEntranceName(destList[locIndex].override) : unknown;
+            const char* rplcDstName = gSettingsContext.ingameSpoilers || isDiscovered ? GetEntranceName(destList[locIndex].overrideDestination) : unknown;
+
+            Draw_DrawFormattedString_Small(10, locPosY, color, "%s to %s %c", origSrcName, origDstName, RIGHT_ARROW_CHR);
+            Draw_DrawFormattedString_Small(10, itemPosY, color, "  %s from %s", rplcDstName, rplcSrcName);
+        }

```

Comment:
I tried using a struct array pointer, but it didn't work as intended when reading the values so I ended up duplicating the code instead.